core/frontend/src/internal/tile/RealityTileLoader.ts

```diff
@@ -99,8 +99,16 @@ export abstract class RealityTileLoader {
     const { is3d, yAxisUp, iModel, modelId } = tile.realityRoot;
     let reader: GltfReader | ImdlReader | undefined;
 
+    let ecefTransform: Transform;
+
+    try {
+      ecefTransform = tile.tree.iModel.getEcefTransform();
+    } catch {
+      ecefTransform = Transform.createIdentity();
+    }
+
```

Comment:
```suggestion
    const ecefTransform = tile.tree.iModel.isGeoLocated ? tile.tree.iModel.getEcefTransform() : Transform.createIdentity();
```
Just check the condition that ensures it won't throw, instead of catching a predictable exception.